spec/models/person_spec.rb

```diff
@@ -227,4 +227,23 @@
       end
     end
   end
+
+  describe '#is?' do
+    context 'given another person' do
+      it 'returns false' do
+        other_person = create(:person, email: 'test.user@digital.justice.gov.uk')
+        expect(person.is?(other_person)).to be false
+      end
+    end
+
+    context 'given a person with matching email' do
```

Comment:
Similar to the comment on the `is?` method, this scenario is not realistic as there can't be 2 people with matching email.